Car/TeamNull SmartCar/src/manager/MovementManager.cpp

```diff
@@ -1,63 +1,227 @@
 #include "MovementManager.hpp"
+#include "WifiManager.hpp"
 
 int carSpeedSet = 0;
 int carSpeedActual = 0;
 int turnAngleSet = 0;
 int turnAngleActual = 0;
+int classSafeDistance = 0 ;
 
-//TODO: Calculate gyroscope offset
-int gyroscopeOffset = 0;
+boolean colisionBeingAvoided = false;
+boolean obstacleBeingAvoided = false;
+boolean obstacleAvoidanceTurnedRight = false;
+boolean obstacleAvoidanceTurnedLeft = false;
+long timeObstacleWasAvoided = millis();
+long timereturnToRouteBegan = millis();
 
-void setDesiredVehicleSpeed (int speed) {
+//This is a helper method for logging purposes only
+String booleanToString(boolean boolean)
```

Comment:
It's probably not the best idea to call a variable after a type. Would you ever write `int int`? :)

Reply:
I have seen String string many times, however, I've now renamed this variable in my own branch